monitoring/daily_report.py

```
@@ -0,0 +1,654 @@
+            'chat_id': CONFIG['telegram_chat_id'],
+            'text': message,
+            'parse_mode': 'HTML'
+        }).encode('utf-8')
+
+        req = Request(url, data=data, method='POST')
+        req.add_header('Content-Type', 'application/x-www-form-urlencoded')
+
+        with urlopen(req, timeout=10) as response:
+            result = json.loads(response.read().decode('utf-8'))
+            return result.get('ok', False)
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)